ts/features/payments/payment/components/WalletPaymentConfirmContent.tsx

```diff
@@ -116,15 +115,16 @@ export const WalletPaymentConfirmContent = ({
   );
 };
 
-const getPaymentSubtitle = (details: UIWalletInfoDetails): string => {
+const getPaymentSubtitle = (
+  details: UIWalletInfoDetails
+): string | undefined => {
   if (details.maskedPan !== undefined) {
-    return `${format(details.expiryDate, "MM/YY")}`;
+    return undefined;
   } else if (details.maskedEmail !== undefined) {
     return I18n.t("wallet.onboarding.paypal.name");
   } else if (details.maskedNumber !== undefined) {
     return `${details.bankName}`;
   }
-
   return "";
```

Comment:
We could return `undefined` as the default value and remove the `details.maskedPan !== undefined` case. What do you think?

Reply:
Yes, it is better!

Addressed into 6ceaab06d59c5baf67285a992c3ca18c86a0a193